flowmachine/flowmachine/core/union.py

```diff
@@ -14,25 +14,26 @@ class Union(Query):
 
     Parameters
     ----------
-    top, bottom : flowmachine.Query object
-        First and second tables respectively
+    queries : flowmachine.Query object
+        Queries to union
+    all : bool, default: True
+        If True, queries will be deduped, if set to False, duplicate rows will be preserved.
     """
 
-    def __init__(self, top, bottom, all=True):
-        """"""
-
-        self.top = top
-        self.bottom = bottom
+    def __init__(self, *queries: Query, all: bool = True):
```

Comment:
Is it worth sorting the list of queries (either sorting in `__init__`, or doing `self.queries = set(queries)` and sorting in `_make_query`)? That way, the query (and query ID) is the same regardless of the input order.

Reply:
That could mess up any queries that depend on an already-derived ordering prior to joining (which shouldn't happen, but could)

Reply:
I think not because the order is meaningful when deduping (e.g. I've ordered home locations by time).